Alignment/OfflineValidation/macros/FitPVResiduals.C

```diff
@@ -290,14 +291,17 @@ void setStyle();
 
 ofstream outfile("FittedDeltaZ.txt");
 
-const Int_t nBins_  = 48;
+Int_t nBins_  = 48;
+const Int_t nPtBins_ = 48;
 Float_t _boundMin   = -0.5;
 Float_t _boundSx    = (nBins_/4.)-0.5;
 Float_t _boundDx    = 3*(nBins_/4.)-0.5;
 Float_t _boundMax   = nBins_-0.5;
 Float_t  etaRange   = 2.5;
 bool     isDebugMode = false;
 
+Double_t mypT_bins[49] = {0.5,0.6,0.7,0.8,0.9,1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2.0,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3.0,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4.0,4.1,4.25,4.5,4.75,5.0,5.5,6.,7.,8.,9.,11.,14.,20.}; 
```

Comment:
@arunhep: no particular reason to prefer float over double, though maybe better to use native C++ types instead in the EDAnalyzer and keep ROOT CINT types here. As for the binning I chose one similar to the one used in CMS-TRK-10-005, paragraph 3.2 (keeping into account the pT spectrum is steeply falling). Do you have a counter-proposal?